gradlew

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
+# Copyright 2015 the original author or authors.
```

Comment:
Could you change this to `Copyright 2015-2020`

Reply:
This script is auto-generated by `./gradlew wrapper --gradle-version 6.7 --distribution-type bin` so that should not be changed.

Reply:
FYI I just updated the wrapper just now from version `6.6.1` to `6.7`.